### PR TITLE
Chenmo/add how long when where next meeting

### PR DESCRIPTION
**Pull request by Theodosia1005**
## Description

Add ask details to next event flow.
Add ask parameter entity to check which detail should we give user

## Testing Steps

When and where is my next meeting?
What's my next meeting?

## Checklist
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the appropriate unit tests
- [ ] I have updated related documentation

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
- [ ] A duplicate issue is filed to track future  work.

If you have updated responses or `.lu` files:
- [x] All languages have been updated
- [x] You have tested deployment with your new models

### CODE REVIEW

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs [hunk 1]
```diff
@@ -41,6 +41,7 @@ public CalendarSkillState()
             ReadOutEvents = new List<EventModel>();
             Duration = 0;
             MoveTimeSpan = 0;
+            AskParameterContent = "";
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs]: string.Empty

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs [hunk 2]
```diff
@@ -162,6 +165,7 @@ public void Clear()
...
```
Comment (lzc850612) [inline solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs]: string.Empty

**Review by lzc850612 — COMMENTED** [review]
Did we run the .lu file change with the pre-built team?

**Comment by Theodosia1005** [comment]
Not yet. This change can use the old Luis model and it will act like the old flow also. We are considering merge all flows of asking for events to one flow. For know, We got Get Summary flow and Get Next Meeting flow at the same time. It can be kind of confuse. We will discuss this with the new flow designer Rachel later on all these then update all the related intents and entities and talk to pre-build team later.

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs (hunk 1 above)
Comment (Theodosia1005) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs]: Done

File: solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs (hunk 2 above)
Comment (Theodosia1005) [reply solutions/Virtual-Assistant/src/csharp/skills/calendarskill/CalendarSkillState.cs]: Done

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**